include/irr/asset/IMeshPacker.h

```diff
@@ -0,0 +1,123 @@
+#ifndef __IRR_I_MESH_PACKER_H_INCLUDED__
+#define __IRR_I_MESH_PACKER_H_INCLUDED__
+
+namespace irr
+{
+namespace asset
+{
+
+namespace meshPackerUtil
```

Comment:
don't invent a new namespace, this is what a non-templated Interface/Base class is for

Reply:
STILL

Reply:
I want  to focus on other things right now, is it ok if I take care of it later?

Reply:
sure, just dont forget